projects/igniteui-angular/src/lib/list/list.common.ts

```diff
@@ -20,6 +22,15 @@ export class IgxListBaseDirective extends DisplayDensityBase {
     children: QueryList<any>;
     listItemLeftPanningTemplate: IgxListItemLeftPanningTemplateDirective;
     listItemRightPanningTemplate: IgxListItemRightPanningTemplateDirective;
+    private _resourceStrings = CurrentResourceStrings.ListResStrings;
+
+    set resourceStrings(value: IListResourceStrings) {
+        this._resourceStrings = Object.assign({}, this._resourceStrings, value);
+    }
+
+    get resourceStrings(): IListResourceStrings {
+        return this._resourceStrings;
+    }
```

Comment:
You should move this to `IgxListComponent`. Also resourceStrings should be a public input and should have an API docs.